File: src/main/java/io/junseok/mealmate/domain/restaurant/dto/request/RestaurantRegister.java
package io.junseok.mealmate.domain.restaurant.dto.request;

import java.time.LocalDateTime;

public record RestaurantRegister(
    String restaurantName,
    String restaurantType,
    String restaurantNotice,
    String location,
    String telNum,
    String openAt,
    String closeAt) {

}

File: src/main/java/io/junseok/mealmate/domain/restaurant/dto/request/RestaurantRegister.kt
package io.junseok.mealmate.domain.restaurant.dto.request;

data class RestaurantRegister(
    val restaurantName: String,
    val restaurantType: String,
    val restaurantNotice: String,
    val location: String,
    val telNum: String,
    val openAt: String,
    val closeAt: String
)
